### ISSUE TITLE
[Bug]: Appsec warning: Remote config path too long

### ISSUE THREAD

**Issue by nclavaud** [DataDog/dd-trace-php#2892]
### Bug report

Hi, after updating the tracer from `1.3.2` to `1.4.0`, our logs show a log of entries saying "remote config path too long":
```
PHP message: PHP Warning:  PHP Request Startup: [ddappsec] Remote config path too long: /ddrc1695****-****************5971 in Unknown on line 0
```

And indeed the remote path here counts `33` characters (not sure if needed but I obfuscated the ID before posting), which exceeds the limit `MAX_LENGTH_OF_REM_CFG_PATH 31` defined in file [`appsec/src/extension/request_lifecycle.c`](https://github.com/DataDog/dd-trace-php/blob/1c81b25efbe5ae617fea6477b8a378dcad8a393f/appsec/src/extension/request_lifecycle.c).

I can see this is related to https://github.com/DataDog/dd-trace-php/pull/2864 cc @cataphract @Anilm3 
but I don't know if there is anything wrong with our remote config path (where is it defined? we use the heroku buildpack) or if the limit is too low?

### PHP version

8.3.12

### Tracer or profiler version

1.4.1

### Upgrading from

Upgrading `dd-trace-php` from `1.3.2` to `1.4.0`.

**Comment by bwoebi** [comment]
Hey @nclavaud,

Do I see that correctly that the number before the dash is 8 characters long? I.e. you have uids greater than `99999`?
I don't think we have tested such a scenario.

**Comment by nclavaud** [comment]
> Do I see that correctly that the number before the dash is 8 characters long?

@bwoebi yes, the path in the logs has 8 digits before the dash and 19 or 20 digits after the dash.

More examples:
```
Remote config path too long: /ddrc41456195-*****************22
Remote config path too long: /ddrc20196849-*****************835
```

> I.e. you have uids greater than 99999?

I don't know? Where does this ID come from?

**Comment by bwoebi** [comment]
These IDs are the effective user ids the php processes run under.

I'll soon provide a fix to encode that hash after the dash as base64 instead of decimal. That should shorten it enough.

**Comment by bwoebi** [comment]
@nclavaud This has been fixed in just now released 1.4.2. Thanks :-)

**Comment by nclavaud** [comment]
@bwoebi Thanks to you! 🙌  I can confirm we have no more ddappsec warnings in our logs with the 1.4.2 release 🥳
